### DISCUSSION TITLE
Enhancement: make num1.png not look like a 7

### DISCUSSION THREAD

**Discussion by arceuss** [FunkinCrew/Funkin#1620] [Ideas]
its in the name

**Comment by exurd** [comment]
### Side-by-side comparison:
![1??](https://raw.githubusercontent.com/ninjamuffin99/Funkin/2a559f17509b70bb97d95714ef33451ba9fe9f36/assets/preload/images/num7.png) ![7??](https://raw.githubusercontent.com/ninjamuffin99/Funkin/2a559f17509b70bb97d95714ef33451ba9fe9f36/assets/preload/images/num1.png)

**Comment by arceuss** [comment]
> ### Side-by-side comparison:
> 
> ![1??](https://raw.githubusercontent.com/ninjamuffin99/Funkin/2a559f17509b70bb97d95714ef33451ba9fe9f36/assets/preload/images/num7.png) ![7??](https://raw.githubusercontent.com/ninjamuffin99/Funkin/2a559f17509b70bb97d95714ef33451ba9fe9f36/assets/preload/images/num1.png)

yeah but it can still look like a 7 to some people

**Comment by exurd** [comment]
I was agreeing with you. It does look a bit too similar.

**Comment by arceuss** [comment]
> 
> 
> I was agreeing with you. It does look a bit too similar.

ah
it was just a misunderstanding then
